### PR TITLE
Update template-glib to 3.35

### PR DESCRIPTION
**Pull request by taozuhong**

### CODE REVIEW

File: mingw-w64-template-glib/PKGBUILD [hunk 1]
```diff
@@ -13,14 +13,15 @@ depends=("${MINGW_PACKAGE_PREFIX}-glib2"
 makedepends=("${MINGW_PACKAGE_PREFIX}-meson"
              "${MINGW_PACKAGE_PREFIX}-ninja"
              "${MINGW_PACKAGE_PREFIX}-pkg-config"
-             "${MINGW_PACKAGE_PREFIX}-cc"
+             "${MINGW_PACKAGE_PREFIX}-gcc"
```
Comment (Biswa96) [inline mingw-w64-template-glib/PKGBUILD:16]: This should be cc

Comment (taozuhong) [reply mingw-w64-template-glib/PKGBUILD:16]: image

Comment (Biswa96) [reply mingw-w64-template-glib/PKGBUILD:16]: That is for mingw64, mingw32 and ucrt64 environments. For clang64 and clang32 cc aliased to clang.

Comment (taozuhong) [reply mingw-w64-template-glib/PKGBUILD]: Done, thanks.

**Comment by Biswa96** [comment]
Thank you for the contribution ❤️

**Pull request merged**